src/test/resources/com/puppycrawl/tools/checkstyle/astprinter/InputJavadocComment.javadoc

```diff
@@ -1,4 +1,4 @@
 <p>
 My {@code Class}
 </p>
-@see never
\ No newline at end of file
+@see never
```

Comment:
This because of the _predicate_ (`_input.LA(1) == '.' || ...`) I've added after `PACKAGE` token. I haven't added anything like `_input.LA(1) == EOF` in there since that is an impractical case for actual java files, so I added `NEWLINE` before EOF there so that `never` can get recognized as PACKAGE as `Character.isWhitespace(_input.LA(1))` in the predicate would pass for `NEWLINE`. I've explained the reason for adding the predicate [below](https://github.com/checkstyle/checkstyle/pull/4671#discussion_r126317644)